### ISSUE TITLE
Set AOI to AOI downloaded from eBAGIS - surface.gdb is corrupted

### ISSUE THREAD

**Issue by jdduh** [PSU-CSAR/vb-bagis#53]
This is done in ArcGIS 10.4.1.

I got this error message when accessing the AOI downloaded from eBAGIS the first time. 

![image](https://user-images.githubusercontent.com/11510708/29789058-b486b2d6-8be9-11e7-8996-1369741f1040.png)

Subsequently, when using the AOI utilities tool to set the AOI to any AOI, I got this error message. 

![image](https://user-images.githubusercontent.com/11510708/29789243-46444378-8bea-11e7-8c6b-0cd8ee2ca3ac.png)

After restarting ArcMap, the problem went away and everything is back to normal until the next time I set the AOI to an AOI that is freshly downloaded from eBAGIS.

**Comment by jdduh** [comment]
Here is the error message in a text file.
[set_aoi_utilities_error.txt](https://github.com/PSU-CSAR/vb-bagis/files/1257918/set_aoi_utilities_error.txt)

**Comment by lbross** [comment]
I have encountered the surface.gdb error before, even on AOI's that I work with every day. I don't think this has anything to do with eBAGIS. The subroutine that generated the error initially uses classes from the GeoAnalyst ArcObjects library which seems to work less reliably with each new version of ArcMap.

I found that we are not explicitly setting the outWorkspace in the rasterAnalysis environment. I will fix this and maybe it will help.

It seems that once the subroutine fails once, it continues to fail which is what generated the ongoing exception and crash. From the stacktrace you provided, I can add some error handling that will prevent the crash but I don't know how well BAGIS will continue to function avoiding the crash. It won't be able to access the dem info.
